app/views/_create-secret.html

```diff
@@ -0,0 +1,267 @@
+<!-- Error messages -->
+<parse-error error="error" ng-if="error"></parse-error>
```

Comment:
yes it is... when creating a secret and invalid input is entered, eg: secret name that already exists.
![screenshot-37](https://cloud.githubusercontent.com/assets/1668218/18549917/ad3e66a8-7b51-11e6-9ddb-216206ff8f30.png)